vsintegration/src/FSharp.Editor/DocumentHighlights/DocumentHighlightsService.fs

```diff
@@ -51,16 +49,17 @@ type internal FSharpDocumentHighlightsService [<ImportingConstructor>] (checkerP
         |> Seq.distinctBy (fun span -> span.TextSpan.Start)
         |> Seq.toArray
 
-    static member GetDocumentHighlights(checker: FSharpChecker, document: Document, position: int, 
-                                        defines: string list, options: FSharpProjectOptions, languageServicePerformanceOptions: LanguageServicePerformanceOptions) : Async<FSharpHighlightSpan[] option> =
+    static member GetDocumentHighlights(document: Document, position: int) : Async<FSharpHighlightSpan[] option> =
         asyncMaybe {
-            let! sourceText = document.GetTextAsync() |> liftTaskAsync
-            let filePath = document.FilePath
+            let! symbol = document.TryFindFSharpLexerSymbolAsync(position, SymbolLookupKind.Greedy, false, false, nameof(FSharpDocumentHighlightsService.GetDocumentHighlights))
```

Comment:
Stylistically I think I prefer binding `userOpName` at the top of the scope like before but it's no big deal.